client/gutenberg/editor/utils.js

```diff
@@ -3,29 +3,37 @@
  * External dependencies
  */
 import apiFetch from '@wordpress/api-fetch';
-import { noop } from 'lodash';
-
-/**
- * Internal dependencies
- */
-import debugFactory from 'debug';
-
-const debug = debugFactory( 'calypso:gutenberg' );
+import wpcomProxyRequest from 'wpcom-proxy-request';
+import { noop, startsWith } from 'lodash';
```

Comment:
fun fact, we should be able to use `String.prototype.startsWith` now that we are including the core-js transform - right @ockham ?

Reply:
Thanks @flootr, appreciate the drive-by comment :smile: 

I'd be fine with phasing out lodash functionality in favor of 'native' ES6(+) (unless there are performance concerns for the Babel polyfills -- there might have been in the past IIRC).

As for type concerns -- I'm enough of a static typing fan that I'd consider using `startsWith` on an integer as plain wrong (and somewhat optimistic that we aren't that doing anywhere in our codebase). Tolerance for `null` and `undefined` might be arguably nice, but lack thereof shouldn't be a dealbreaker.

> ```js
> obj.d && typeof obj.d === 'string' && obj.d.startsWith( '1' );
> ```

Admittedly, that's kinda ugly. Destructuring might make it a bit nicer maybe?